src/pal/src/safecrt/internal_securecrt.h

```diff
@@ -23,6 +23,13 @@
 #ifndef _INC_INTERNAL_SECURECRT
 #define _INC_INTERNAL_SECURECRT
 
+#ifndef __GLIBC__    // e.g. OS with musl-libc
+typedef unsigned long __uint64_t;
+typedef signed long __int64_t;
+typedef __int64_t int64_t;
+typedef __uint64_t uint64_t;
```

Comment:
I tried adding `#include <inttypes.h>` header and that collided with other typedefs such as:

```ash
[  3%] Building CXX object src/pal/src/CMakeFiles/coreclrpal.dir/objmgr/shmobject.cpp.o
[  3%] Building CXX object src/pal/src/CMakeFiles/coreclrpal.dir/objmgr/shmobjectmanager.cpp.o
[  3%] Building C object src/pal/src/CMakeFiles/coreclrpal.dir/safecrt/makepath_s.c.o
In file included from /root/coreclr/src/pal/src/safecrt/makepath_s.c:20:
In file included from /root/coreclr/src/pal/src/safecrt/mbusafecrt_internal.h:22:
/root/coreclr/src/pal/inc/pal_mstypes.h:224:16: error: typedef redefinition with different types
      ('char' vs 'signed char')
typedef __int8 int8_t;
               ^
/usr/include/bits/alltypes.h:137:25: note: previous definition is here
typedef signed char     int8_t;
                        ^
1 error generated.
```